Snakemake/workflow/rules/ancestral_inference.smk

```diff
@@ -7,10 +7,10 @@ if config['ancestral_allele'] == None:
             alignedFocal=config['aligned_focal'],
             vcf=config['raw_vcf']
         output:
-            info=f"../{Project}/AncestralAllele/Raw_vcf_info.INFO", 
-            log=temp(f"../{Project}/AncestralAllele/Raw_vcf_info.log")
+            info=f"{workdir}/AncestralAllele/RawVcfInfo.INFO",
+            log=temp(f"{workdir}/AncestralAllele/RawVcfInfo.log")
         params:
-            prefix=f"../{Project}/AncestralAllele/Raw_vcf_info"
+            prefix=f"{workdir}/AncestralAllele/RawVcfInfo"
         conda: "bcftools"
```

Comment:
@IsidoreHouaga , there is a change here in the file names from the previous version - it should be "Raw_vcf_info". Put the paths look good :)